src/Altinn.App.Core/Models/UserAction/FrontendAction.cs

```diff
@@ -0,0 +1,33 @@
+namespace Altinn.App.Core.Models.UserAction;
+
+/// <summary>
+/// Defines an action that should be performed by frontend
+/// </summary>
+public class FrontendAction
+{
+    /// <summary>
+    /// Name of the action
+    /// </summary>
+    public string Name { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Metadata for the action
+    /// </summary>
+    public Dictionary<string, object>? Metadata { get; set; }
+    
+    /// <summary>
+    /// Creates a gotoPage action
+    /// </summary>
+    /// <param name="frontendPage"></param>
+    /// <returns></returns>
+    public static FrontendAction GotoPage(string frontendPage)
+    {
+        var frontendAction = new FrontendAction()
+        {
+            Name = "gotoPage",
```

Comment:
Is this the one we need now, or is it navigateNextPage?

Reply:
Correct we need the navigateToNextPage, will replace gotoPage with navigateToNextPage.

Reply:
After having a quick look at the frontend issue the correct name is `nextPage`